graylog2-web-interface/src/components/configurations/SearchesConfig.jsx

```diff
@@ -258,27 +154,32 @@ const SearchesConfig = React.createClass({
                    checked={this._isEnabled()}
                    onChange={this._onChecked}/>
             {this._isEnabled() &&
-            <Input type="text"
-                   ref="query_time_range_limit"
-                   label="Query time range limit (ISO8601 Duration)"
-                   onChange={this._onPeriodUpdate('query_time_range_limit')}
-                   value={config.query_time_range_limit}
-                   help={'The maximum time range for searches. (i.e. "P30D" for 30 days, "PT24H" for 24 hours)'}
-                   addonAfter={this._formatDuration(this.state.config.query_time_range_limit)}
-                   bsStyle={this._validationState()}
-                   required/>
+            <ISODurationInput duration={config.query_time_range_limit}
+                              update={this._onUpdate('query_time_range_limit')}
+                              label="Query time range limit (ISO8601 Duration)"
+                              help={'The maximum time range for searches. (i.e. "P30D" for 30 days, "PT24H" for 24 hours)'}
```

Comment:
Ah, didn't see that. I think it's fine then, otherwise eslint would complain about it. I guess the lesson to learn is that there's always someone complaining :smile: